api/app/signals/apps/services/domain/pdf_summary.py

```diff
@@ -161,6 +161,7 @@ def _get_context_data(signal, user, include_contact_details):
             'now': timezone.now(),
             'reporter_email': reporter_email,
             'reporter_phone': reporter_phone,
+            'FEATURE_FLAGS': settings.FEATURE_FLAGS,
```

Comment:
We have trying to reduce the amount of information available in the email context to a minimum. I am not sure that exposing the full `FEATURE_FLAGS` object is desirable - because most of the information therein is not (ever) relevant in an email to a reporter.

Consider adding only `SHOW_REPORTER_CONTACT_DETAILS_IN_PDF` to the email context.

Reply:
Followed the approach used in context_processors.py

Reply:
I'll close this conversation, as the PDF template is not editable by the users (unlike emails that I erroneously mentioned). Your implementation is fine.